docs/pages/reference/resources.mdx

```diff
@@ -102,6 +103,44 @@ Roles govern access to databases, SSH servers, Kubernetes clusters, web services
 
 (!docs/pages/includes/role-spec.mdx!)
 
+## Windows Desktop
+
+In most cases, Teleport will register `windows_desktop` resources automatically
+based on static hosts in your configuration file or via LDAP-based discovery.
+
+However, you can also manage `windows_desktop` resources manually with `tctl`.
+This can be useful for managing inventories of hosts that are not joined to
+an Active Directory domain.
+
+There are a few important considerations to keep in mind when registering
+desktops this way:
+
+1. The desktop's `addr` can be a hostname or IP address, and should include
+   the RDP port (typically 3389).
+1. The desktop's `host_id` should be set to the name of a
```

Comment:
name or id ? I think name is a bit misleading here.

Reply:
The name of the w_d_s is the host_id of the desktop.

```
➜ tctl get windows_desktop_service
kind: windows_desktop_service
metadata:
  expires: "2023-09-15T17:02:32.737179Z"
  id: 1694796752737618000
  name: 307e091b-7f6b-42e0-b78d-3362ad10b55d
spec:
  addr: 127.0.0.1:3028
  hostname: mbp16
  teleport_version: 15.0.0-dev
version: v3
```